wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -34,8 +34,8 @@
     from wandb.sdk.artifacts.artifact_manifest_entry import ArtifactManifestEntry
     from wandb.sdk.internal import progress
 
-# This makes the first sleep 1s, and then doubles it up to total times,
-# which makes for ~18 hours.
+# Sleep length: 0, 2, 4, 8, 16, 32, 64, 120, 120, 120, 120, 120, 120, 120, 120, 120
```

Comment:
To me at least it's actually a little clearer with words, e.g. "doubles (plus jitter) until it reaches the maximum of 2 minutes"

Reply:
It's a little more complicated than that and I didn't want to repeat the documentation.